Packs/CortexXDR/Integrations/CortexXDRIR/test_data/get_incident_extra_data_new_status.json

```diff
@@ -1,5 +1,8 @@
 {
     "reply": {
+        "incidents":
```

Comment:
What are those changes meant to represent? the response from the API is different than what we thought?

Reply:
I fixed the format, it suppose to be as mentioned here: 
https://cortex-panw.stoplight.io/docs/cortex-xdr/a61eadc13dd54-get-all-incidents